enterprise/utils.py

```diff
@@ -512,3 +512,17 @@ def safe_extract_key(data, key, default=''):
         return data[key]
     else:
         return default
+
+
+def filter_audit_course_modes(enterprise_customer, course_modes):
+    """
+    Filter audit course modes out if the enterprise customer has the 'Hide audit track' flag enabled.
+
+    Arguments:
+        enterprise_customer: The EnterpriseCustomer that the enrollment was created using.
+        course_modes: iterable with dictionaries containing a required 'mode' key
+    """
+    if not enterprise_customer.hide_audit_track:
+        return course_modes
+    audit_modes = getattr(settings, 'ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES', ['audit'])
```

Comment:
@mattdrayer I created this [PR](https://github.com/edx/edx-platform/pull/15095) to add the setting to the common and aws modules.

Could you please [review this commit](https://github.com/replaceafill/configuration/commit/98b70ea90a1dfd8feb5f235c8602f7e73d58d318) to add the setting to the configuration repository and recommend how could I test that?